### PR TITLE
Use the correct notation for getting documents

### PR DESCRIPTION
**Pull request by brunoocasali**
Fix an issue pointed initially here https://github.com/meilisearch/meilisearch-php/issues/353

### CODE REVIEW

File: tests/Search/SearchResultTest.php [hunk 1]
```diff
@@ -229,4 +229,25 @@ public function testTransformFacetsDistributionMethod(): void
         $this->assertEquals(1, $response->getFacetDistribution()['genre']['FANTASY']);
         $this->assertEquals(1, $response->getFacetDistribution()['genre']['ADVENTURE']);
     }
+
+    public function testRonaldo(): void
```
Comment (alallema) [inline tests/Search/SearchResultTest.php:233]: Why this name of test?

File: src/Contracts/IndexesResults.php [hunk 2]
```diff
@@ -20,7 +20,7 @@ public function __construct(array $params)
     }
 
     /**
-     * @return array<int, array>
+     * @return array<int, Indexes>
```
Comment (alallema) [inline src/Contracts/IndexesResults.php:23]: Php stan still doesn't seem happy with this, it is normal?

**Review by alallema — COMMENTED** [review]
I'm not sure to fully understand the changes here

File: src/Contracts/IndexesResults.php (hunk 2 above)
Comment (norkunas) [reply src/Contracts/IndexesResults.php:23]: Because the `Indexes` import is missing :)

Comment (norkunas) [reply src/Contracts/IndexesResults.php:23]: Also not sure why introducing it here if I've already made a PR for this fix? :D

File: tests/Search/SearchResultTest.php (hunk 1 above)
Comment (brunoocasali) [reply tests/Search/SearchResultTest.php]: It was a mistake it shouldn't be commited, nevermind 🤦

**Review by alallema — APPROVED** [review]
LGTM! 🎉

**Comment by brunoocasali** [comment]
bors merge

**Pull request merged**